DNSecure/Models/Presets.swift

```diff
@@ -44,10 +44,10 @@ enum Presets {
                     servers: [
                         "1.1.1.1",
                         "1.0.0.1",
-                        "2606:4700:4700::64",
-                        "2606:4700:4700::6400",
+                        "2606:4700:4700::1111",
+                        "2606:4700:4700::1001",
                     ],
-                    serverName: "cloudflare-dns.com"
+                    serverName: "1dot1dot1dot1.cloudflare-dns.com"
```

Comment:
Do you have any reference to this?

According to [the Apple's documentation](https://developer.apple.com/documentation/networkextension/nednsovertlssettings/3552332-servername), `serverName` will be used for TLS validation. However, `1dot1dot1dot1.cloudflare-dns.com` is just a host name of `1.1.1.1` (and `1.0.0.1`), so I believe that the old one is correct for that purpose.

Reply:
According to 1.1.1.1 [#2](https://developers.cloudflare.com/1.1.1.1/encryption/dns-over-tls/), DNS over TLS is available on `1.1.1.1` or `1.0.0.1` or `1dot1dot1dot1.cloudflare-dns.com` or `one.one.one.one`. `cloudflare-dns.com` did not seem to work.

Reply:
`1dot1dot1dot1.cloudflare-dns.com` and `cloudflare-dns.com` are servers with different purposes. `1dot1dot1dot1.cloudflare-dns.com` is the domain name for 1.1.1.1 and 1.0.0.1. On the other hand, `cloudflare-dns.com` is the server for the TLS validation.

Reply:
I do not know much about DNS but `cloudflare-dns.com` does not seem to work while `1dot1dot1dot1.cloudflare-dns.com` works when used as DoT servers without other servers.

Reply:
You are correct that cloudflare-dns.com is not a DoT server. DoT servers must be specified in `servers`, and the `serverName` must be a server for TLS validation not a DoT server.

```swift
                DoTConfiguration(
                    servers: [
                        "1.1.1.1",
                        "1.0.0.1",
                        "2606:4700:4700::64",
                        "2606:4700:4700::6400",
                    ],
                    serverName: "cloudflare-dns.com"
                )
```

From https://developer.apple.com/documentation/networkextension/nednsovertlssettings/3552332-servername,

> The server will be accessed over TCP port 853, as defined in [RFC 7858](https://tools.ietf.org/html/rfc7858). The server name is used for TLS validation.

Reply:
In case of CloudFlare, they uses cloudflare-dns.com as `tls-host` in this "Example" section.

https://developers.cloudflare.com/1.1.1.1/encryption/dns-over-tls/#example

Reply:
I do not know so I reverted it.

Reply:
Thanks.